mlrun/runtimes/databricks_job/databricks_runtime.py

```diff
@@ -79,6 +82,60 @@ def _pre_run(self, runspec: RunObject, execution):
         else:
             raise ValueError("Databricks function must be provided with user code")
 
+    def run(
```

Comment:
I wonder if we can't just forward args and kwargs? This is really too much error prone boilerplate.

Reply:
I guess that would mess up autocomplete so maybe better just keep this fluff.